ratpack-core/src/main/java/ratpack/http/internal/DefaultStatus.java

```diff
@@ -41,4 +41,9 @@ public String getMessage() {
   public HttpResponseStatus getNettyStatus() {
     return status;
   }
+
+  @Override
+  public String toString() {
+    return status.code() + ":" + status.reasonPhrase();
```

Comment:
This should probably cater for the reason phrase to be empty as it not required to be set.